circle.yml

```diff
@@ -35,6 +35,7 @@ test:
     override:
         - bundle exec rake lint
         - rake ci:run[default]
+        - rake ci:run[burrow]
```

Comment:
could you add this line to the `.travis.yml` file instead? We're not using circleCI anymore